test/e2e/flowexporter_test.go

```diff
@@ -31,10 +31,8 @@ import (
 // TestFlowExporter runs flow exporter to export flow records for flows.
 // Flows are deployed between Pods on same node.
 func TestFlowExporter(t *testing.T) {
-	// TODO: remove this limitation after flow_exporter supports IPv6
-	skipIfIPv6Cluster(t)
-	skipIfNotIPv4Cluster(t)
-	data, err := setupTestWithIPFIXCollector(t)
+	// Should I add skipBenchmark as this runs iperf?
+	data, err, isIPv6 := setupTestWithIPFIXCollector(t)
```

Comment:
so we now have test coverage for IPv4, IPv6 and dual-stack?

Reply:
Mostly. Except that I don't have a good idea how to test the case you mentioned:
An `ipfix-collector` is outside of a dual-stack cluster and its IP family is different from the main IP family (same as node's internal-ip) of the cluster. Pods generated in e2e should be in a cluster and `ipfix-collector` Pod uses host network so it only has one IP.

Reply:
@lzhecheng no worries about this. Right now we always deploy the ELK stack inside the cluster and the aggregator will of course be deployed in the cluster. I just wanted to make sure that we didn't make any assumption as to the address family of the collector and looks like it is the case now.